File: ts/wam/7028.ts
export const WamWebArkoseToken = 7028 as const

export interface WebArkoseTokenEvent {
  /** field 1, wire `token` */
  token?: string
  /** field 2, wire `md_session_id` */
  mdSessionId?: string
  /** field 3, wire `md_client_session_id` */
  mdClientSessionId?: string
}

File: go/wam/7028.go
const WamWebArkoseToken = 7028 // channel: regular

type WebArkoseTokenEvent struct {
	Token string `wam:"token"` // field 1
	MdSessionId string `wam:"md_session_id"` // field 2
	MdClientSessionId string `wam:"md_client_session_id"` // field 3
}